dotCMS/src/main/java/com/dotcms/util/ConversionUtils.java

```diff
@@ -213,4 +214,17 @@ public static boolean toBoolean(final String strBool, final boolean defaultBool)
 		}
 	}
 
+	/**
+	 * Based on a value get from the database, if it is a boolean will returns a cast.
```

Comment:
"Based on a value obtained from database, if it is a boolean will return a cast"

Reply:
thx